superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts

```diff
@@ -23,11 +23,11 @@ describe('AdhocFilters', () => {
     cy.route('GET', '/superset/explore_json/**').as('getJson');
     cy.route('POST', '/superset/explore_json/**').as('postJson');
     cy.route('GET', '/superset/filter/table/*/name').as('filterValues');
-  });
-
-  it('Should not load mathjs when not needed', () => {
     cy.visitChartByName('Boys'); // a table chart
     cy.verifySliceSuccess({ waitAlias: '@postJson' });
+  });
+
+  xit('Should not load mathjs when not needed', () => {
```

Comment:
@ktmud I had to disable this, as apparently the function annotation layer that was added to the ECharts Timeseries viz loads `mathjs` despite not being used here. The relevant code is here (is called by `transformProps.ts`): https://github.com/apache-superset/superset-ui/blob/master/plugins/plugin-chart-echarts/src/utils/annotation.ts

Reply:
You chart plugin seems to be using a different version of `mathjs`, which is probably incompatible with what's used in `AnnotationLayer` control. Do you mind changing them to the same version and see if the test still fails?

![image](https://user-images.githubusercontent.com/335541/98718448-50c80980-2343-11eb-9c25-f4769c46aeac.png)

Reply:
Ok, good to know, I'll do that 👍

Reply:
@ktmud I've now upgraded all `mathjs` deps to `8.0.1`, and still failing the test. I propose disabling it for now so we can merge this.

Reply:
The test is failing because of this import chain: `@superset-ui/plugin-chart-echarts -> Timeseries -> transformProps -> utils/annotation -> mathjs`. Unlike `loadChart`, `transformProps` is not loaded asynchronously, so Webpack could not defer the loading of `mathjs`, which means it will be loaded for all first-visits on almost all pages, undoing part of the optimization we did in #10837 .

The test is there to make sure this optimization still works. It didn't break because it was flaky. It caught the breaking changes it is supposed to catch.

It's OK to merge this PR as is for now, but I'd recommend moving the `mathjs` logic to the chart renderer so that we could continue benefiting from asynchronous script loading.

Reply:
Thanks for the context. I'll move the `mathjs` (and any other similar) rendering logic out of `transformProps` and re-enable the test in a subsequent bump.